server_src/routes/system.js

```diff
@@ -16,43 +16,37 @@ const { fetchMongoDBConnectionString } = require("../app-env");
 const { isPm2, isNodemon, isNode } = require("../utils/env.utils.js");
 const { SettingsClean } = require("../lib/dataFunctions/settingsClean");
 
-router.get(
-  "/",
-  ensureAuthenticated,
-  ensureCurrentUserAndGroup,
-  async (req, res) => {
-    const clientSettings = await SettingsClean.returnClientSettings();
-    const serverSettings = await SettingsClean.returnSystemSettings();
-    const systemInformation = await SystemRunner.querySystemInfo();
-    const printers = Runner.returnFarmPrinters();
-    const softwareUpdateNotification =
-      softwareUpdateChecker.getUpdateNotificationIfAny();
-    let dashboardSettings =
-      clientSettings?.dashboard || getDefaultDashboardSettings();
+router.get("/", ensureAuthenticated, ensureCurrentUserAndGroup, async (req, res) => {
+  const clientSettings = await SettingsClean.returnClientSettings();
```

Comment:
I will have to manually copy this to the awilix branch.

Reply:
I think this is prettier doesn't look like an actual change